### PR TITLE
TMA-1182: UFB extended spec passing

### PR DESCRIPTION
**Pull request by kubamahnert**

### CODE REVIEW

File: spec/lcm/integration/user_provisioning_extended_spec.rb [hunk 1]
```diff
@@ -9,40 +9,28 @@
...
-        labels: [
-          {
-            column: 'value',
-            label: extra_mufs.first[:label_id]
-          }
-        ]
-      }
-      extra_project.add_data_permissions GoodData::UserFilterBuilder.get_filters(GoodData::LCM2.convert_to_smart_hash(extra_mufs),
-                                                                                 GoodData::LCM2.convert_to_smart_hash(builder_config))
+      Support::UserProvisioningHelper.test_users_brick(projects: @fixtures[:projects],
```
Comment (yenkins) [inline spec/lcm/integration/user_provisioning_extended_spec.rb]: warning: Similar code found in user_provisioning_extended_spec.rb:20, user_provisioning_extended_spec.rb:23 (mass = 32) (flay)

File: spec/lcm/integration/user_provisioning_extended_spec.rb [hunk 2]
```diff
@@ -9,40 +9,28 @@
...
+                                                       test_context: @fixtures[:brick_params],
+                                                       user_data: @fixtures[:user_data])
+      Support::UserProvisioningHelper.test_user_filters_brick(projects: @fixtures[:projects],
```
Comment (yenkins) [inline spec/lcm/integration/user_provisioning_extended_spec.rb]: warning: Similar code found in user_provisioning_extended_spec.rb:20, user_provisioning_extended_spec.rb:23 (mass = 32) (flay)

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/847/ : SUCCESS in 15s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1660/ : FAILURE in 1m 17s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1921/ : SUCCESS in 1m 14s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/98/ : SUCCESS in 9m 18s

**Comment by kubamahnert** [comment]
https://travis-ci.org/kubamahnert/gooddata-ruby/builds/456982194

**Comment by kubamahnert** [comment]
https://travis-ci.org/kubamahnert/gooddata-ruby/builds/457066475

**Pull request merged**